pkg/flowcli/services/transactions.go

```diff
@@ -132,6 +137,28 @@ func (t *Transactions) send(
 
 	t.logger.StartProgress("Sending transaction...")
 
+	resolver, err := contracts.NewResolver(code)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if resolver.ImportExists() {
+		if network == "" {
+			return nil, nil, fmt.Errorf("missing network, specify which network to use to resolve imports in transaction code")
+		} else if codePath == "" { // when used as lib with code we don't support imports
```

Comment:
I would just make this a separate `if` statement, no need to have an `else` chain if each block has a `return` statement